good-enough-practices-for-scientific-computing.tex

```diff
@@ -685,15 +726,19 @@ \subsection*{Acknowledgments}
     ``scientific guts'' of the project.
 
     The second type of file in \texttt{src} is controller or driver
-    scripts that combine the core analytical functions with particular
-    parameters and data input/output commands in order to execute the
-    entire project analysis from start to finish. A controller script for
+    scripts that is a script that combines all the analysis steps that
```

Comment:
grammar is choppy - try "scripts that combine" instead of "scripts that is a script that combines" ?